lib/modules/manager/bazel-module/bazelrc.ts

```diff
@@ -96,6 +96,36 @@ function createEntry(line: string): BazelrcEntries | undefined {
   return undefined;
 }
 
+export function expandWorkspacePath(
+  value: string,
+  workspaceDir: string,
+): string {
+  if (!value.includes('%workspace%')) {
+    return value;
+  }
+  const absolutePath = upath.resolve(workspaceDir);
```

Comment:
What is a typical value for workspaceDir? Eg is it absolute like /tmp/foo or relative like some/path/?

Reply:
resolved as absolute path